components/dataComponents/card.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+export default function Card({
+  title,
+  text,
+  imagePath,
+}: {
+  title: string
+  text: string
+  imagePath: string
+}) {
+  return (
+    <div
+      className="w-11/12 mt-14 border border-[#FFFFFF]
+    shadow-[0px_0px_4px_rgba(0,0,0,0.25)] box-border boreder-solid rounded-3xl h-96"
```

Comment:
duplicate shadow. move this to the config.
You got typo in border: "boreder"

Reply:
shadow moved to config
fix: boreder -> border